rfc/004-feature-template.md

```diff
@@ -0,0 +1,48 @@
+- Feature Name: badges_node
```

Comment:
hey @allisonsuarez , could you change the rfc name to be 001-support-badge-as-entity.md and change the folder to be rfcs?

Reply:
the doc is still using the template name.